src/pages/home-page/AppLayout.tsx

```diff
@@ -1,12 +1,19 @@
+import React, { useState } from "react";
 import { Outlet } from "react-router-dom";
 import { AppHeader } from "./AppHeader";
 
 export function AppLayout() {
+  const [activeTab, setActiveTab] = useState("all pokemons");
+  const showMyPokemons = activeTab === "my pokemons";
```

Comment:
Use consts and enums for tab titles.
You can create a consts file in the same folder and then you can update there - if someone changes (by mistake) `my pokemons` to `My pokemons` this will break your comparison. A better way would be to use an enum approach like so:
`enum PokemonTabs { ALL = 'All Pokémon'; MINE = 'My Pokémon'}`
and then you use it like so:
`const [activeTab, setActiveTab] = useState(PokemonTabs.ALL);`

or object based approach inside the file (if you don't expect there to be a lot of tabs like so:
`const pokemonTabs = {
  all: { key: 'all', label: 'All Pokémon' },
  mine: { key: 'mine', label: 'My Pokémon' }
}`
and then you use it like so:
`const [activeTab, setActiveTab] = useState(pokemonTabs.all.label);`